### PR TITLE
Feat: Oracle V3 review fixes

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

File: src/providers/execution/contracts/staking_router.py [hunk 1]
```diff
@@ -44,7 +45,22 @@ def get_all_node_operator_digests(self, module: StakingModule, block_identifier:
         """
         Returns node operator digest for each node operator in lido protocol
         """
-        response = self.functions.getAllNodeOperatorDigests(module.id).call(block_identifier=block_identifier)
```
[the remark this answers is not in the document]
Comment (F4ever) [reply src/providers/execution/contracts/staking_router.py:47]: Fixed

File: src/web3py/extensions/lido_validators.py [hunk 2]
```diff
@@ -144,19 +144,22 @@ def get_lido_validators(self, blockstamp: BlockStamp) -> list[LidoValidator]:
...
+
+        return self.merge_validators_with_keys(lido_keys, validators)
+
+    def _kapi_sanity_check(self, keys_count_received: int, blockstamp: BlockStamp):
         no_operators = self.get_lido_node_operators(blockstamp)
 
         # Make sure that used keys fetched from Keys API >= total amount of total deposited validators from Staking Router
         total_deposited_validators = 0
         for node_operator in no_operators:
             total_deposited_validators += node_operator.total_deposited_validators
```
[the remark this answers is not in the document]
Comment (F4ever) [reply src/web3py/extensions/lido_validators.py:157]: Fixed

File: src/variables.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
 LIDO_LOCATOR_ADDRESS: Final = os.getenv('LIDO_LOCATOR_ADDRESS')
 CSM_MODULE_ADDRESS: Final = os.getenv('CSM_MODULE_ADDRESS')
 FINALIZATION_BATCH_MAX_REQUEST_COUNT: Final = int(os.getenv('FINALIZATION_BATCH_MAX_REQUEST_COUNT', 1000))
+EL_REQUESTS_BATCH_SIZE: Final = int(os.getenv('EL_REQUESTS_BATCH_SIZE', 1000))
```
[the remark this answers is not in the document]
Comment (F4ever) [reply src/variables.py:28]: Fixed

File: src/modules/accounting/third_phase/extra_data_v2.py [hunk 4]
```diff
@@ -35,12 +35,12 @@ def collect(
         cls,
         stuck_validators: dict[NodeOperatorGlobalIndex, int],
         exited_validators: dict[NodeOperatorGlobalIndex, int],
-        max_items_count: int,
+        max_items_count_per_tx: int,
```
[the remark this answers is not in the document]
Comment (F4ever) [reply src/modules/accounting/third_phase/extra_data_v2.py:38]: fixed

File: src/services/exit_order_v2/iterator.py [hunk 5]
```diff
@@ -122,7 +133,7 @@ def _calculate_lido_stats(self):
 
             no_validators = len(validators) + transient_validators_count
 
-            self.total_lido_exitable_validators += no_validators
+            self.total_lido_validators += no_validators
             self.module_stats[gid[0]].exitable_validators += no_validators
             self.node_operators_stats[gid].exitable_validators = no_validators
```
[the remark this answers is not in the document]
Comment (F4ever) [reply src/services/exit_order_v2/iterator.py:138]: fixed

**Review by avsetsin — APPROVED** [review]

**Pull request merged**